config/sync/block_content.type.code.yml

```diff
@@ -5,6 +5,6 @@ dependencies: {  }
 _core:
   default_config_hash: iarLrzD0NiTlA4vIQMwEG_nE9sRYK7dK0d5fU9YiTp4
 id: code
-label: Kode
+label: 'Kode (1)'
```

Comment:
This can be used in all but depending on the script or iframe being inserted it might be best to limit it to 1 and 2 columns I think.